### ISSUE TITLE
Proton continues to fail to launch games properly, cannot get past boot.

### ISSUE THREAD

**Issue by dmannorflow** [ValveSoftware/Proton#5566]
After previous success solving my issues with proton, games continue to fail past booting. I have no ideas what could be causing this now, and I do not believe that I have any way to access Proton logs. Linux native games work completely fine, so it must be a proton issue. If anyone has any suggestions for solving this and gaining access to logs, I would gladly take them. 
System: Fedora 35
Specs: Radeon RX570, Ryzen 7

**Comment by kisak-valve** [comment]
Hello @dmannorflow, is this a continuation of #5530 or a separate issue? Proton interacting with NTFS is known to be finicky and will cause trouble more often than not.

**Comment by dmannorflow** [comment]
@kisak-valve it is. However, it is now affecting my OS Drive. I have tried 3 games, and all of them refuse to load.

**Comment by kisak-valve** [comment]
You could be seeing something similar to https://github.com/ValveSoftware/steam-runtime/issues/434. Please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. Also, please check which partition the Proton version you're using and Steam Linux Runtime - Soldier is installed into by going to each of those in Steam's library view, right click on their entry on the left sidebar -> `Manage` -> `Browse local files` and verify that they are not using the Steam library folder on the NTFS partition.

**Comment by dmannorflow** [comment]
@kisak-valve I have checked all versions of proton and Steam Linux Runtine, and they're both on the OS Drive. I'll go ahead and follow the instructions on issue you posted, and I'll give you an update. 
https://gist.github.com/dmannorflow/04db49b0f771b87966608694c1cc459e

**Comment by kisak-valve** [comment]
Thanks, your system information looks fairly healthy. Please list some affected games and  add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, then attach the generated $HOME/steam-$APPID.log to this issue report as a file if they are generated. (Proton logs compress well if needed.)

If there's no logs generated, then please completely close Steam, then run `steam` from a terminal and check if there's any hints in the terminal spew when you tell Steam to start a game that uses Proton.

**Comment by dmannorflow** [comment]
I have found a few errors, wishing I could copy the entire output.

> Gtk-Message: 13:41:42.187: Failed to load module "pk-gtk-module"
> Could not connect to X session manager: None of the authentication protocols specified are supported
> (steam:82575): Gtk-WARNING **: 13:41:48.735: gtk_disable_setlocale() must be called before gtk_init()
> ERROR: ld.so: object '/home/dmannorflow/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
> pid 83666 != 83663, skipping destruction (fork without exec?)

These are all errors that I believe that could have an issue launching proton.

**Comment by ipr** [comment]
Can you give specific examples of games that are not working? Do you know if they are games which are supposed to work yet? Some games have issues that are not supported yet.

> I have found a few errors, wishing I could copy the entire output.
> 
> > Gtk-Message: 13:41:42.187: Failed to load module "pk-gtk-module"
> > Could not connect to X session manager: None of the authentication protocols specified are supported

Like in the previous issue, did you try with Wayland session? For whatever reason, it looks like there are X-related issues and trying out suggestions would eliminate some possibilities.

Side note: you can upload a logfile when making a comment (see the line that says "Attach files.."? click on it and you can select a file to upload)

**Comment by dmannorflow** [comment]
@ipr The games I have tested are Halo MCC, Halo Wars, and Dying Light. I believe that I am running with Wayland session, but I'll double check.

**Comment by dmannorflow** [comment]
@ipr I have tried with Halo MCC, Halo Wars, and Dying Light. I am running on GNOME with Xorg, which I believe is the same thing.

**Comment by ipr** [comment]
> @ipr I have tried with Halo MCC, Halo Wars, and Dying Light. I am running on GNOME with Xorg, which I believe is the same thing.

Didn't you already mention in that other thread you got MCC working when you switched to newer Proton? Did you switch to old Proton again despite it not working?

Xserver and Xwayland are different: Wayland session uses in-kernel (DRM/KMS) drivers which means it doesn't need elevated permissions like old Xserver does (different drivers). So they are different, it is not a matter of believing it or not.

**Comment by dmannorflow** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5566#issuecomment-1038378454

No, I haven't changed anything about Proton since I fixed it. I didn't read much into the differences, so I assumed they were the same. Apologies.

**Comment by dmannorflow** [comment]
@ipr Somehow my logs have started working, and I'll attach them.
[steam-459220.log](https://github.com/ValveSoftware/Proton/files/8056899/steam-459220.log)
[steam-976730.log](https://github.com/ValveSoftware/Proton/files/8056900/steam-976730.log)

**Comment by ipr** [comment]
Both have problems with creating window:
27753.553:00a0:00a4:err:module:process_init failed to load L"C:\\windows\\system32\\explorer.exe", error 4000000e
27753.556:0020:009c:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
27753.556:0020:009c:err:winediag:nodrv_CreateWindow The explorer process failed to start.

^^ This should be using wine's x11 driver but if you are already having issues with X session manager these won't work either.

Wars has some problems with Vulkan:
27754.940:00ac:00c0:err:vulkan:wine_vk_init Failed to load Wine graphics driver supporting Vulkan.
info:  Required Vulkan extension VK_KHR_surface not supported
err:   DxvkInstance: Failed to create instance

^^ This is likely caused by the other failure already happening earlier.

**Comment by aeikum** [comment]
This line in your Steam client output looks very worrying. I don't know what would cause it.

> Could not connect to X session manager: None of the authentication protocols specified are supported

**Comment by dmannorflow** [comment]
@aeikum I have no idea either. Considering wiping and reinstalling Steam to fix it.

**Comment by Hasshu** [comment]
@aeikum I was getting the same log spam on Fedora Linux 35 with GNOME, yet there were no issues. Curiously enough, it doesn't seem to happen on the same OS with KDE Plasma.

@dmannorflow Reinstalling your OS may be a better idea at this point. You also might want to check the SMART data of your drives, as it could be a hardware issue.

**Comment by dmannorflow** [comment]
@Hasshu I highly doubt it's hardware issues, as referenced in a previous post, because I have run Steam under several Debian-based distros, and I have never run into issues with it. It is VR capable, so it isn't underpowered either.

**Comment by RockafellaJaz** [comment]
When I start having issues with steam / proton the first thing a do is try testing a fresh Steam client in a new directory.

- Open a terminal
- Execute 
`mkdir teststeam`
`cd teststeam`
`HOME=$PWD`
 `steam`

This starts a nice and fresh steam client, so there should be nothing local to interfere with it.
Then just log in, adjust your steam play settings, install a windows game that you know should work and try to run it.
If the game runs then the problem is probably caused by something in the current user home directory.
After just delete the test directory.
